app/main/index.js

```diff
@@ -159,7 +159,9 @@ app.on('ready', () => {
 	}
 
 	// Initialize sentry for main process
```

Comment:
We could put this here and re-use at other places-
```
const errorReporting = ConfigUtil.getConfigItem('errorReporting');
```

Reply:
good thought. Will change.